perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -377,7 +391,20 @@ def RunNetperf(vm, benchmark_name, server_ip, num_streams):
                      server_ip=server_ip,
                      length=FLAGS.netperf_test_length,
                      output_selector=OUTPUT_SELECTOR,
-                     confidence=confidence, verbosity=verbosity)
+                     confidence=confidence,
+                     verbosity=verbosity)
+
+  if benchmark_name.upper() == 'UDP_STREAM':
+    netperf_cmd += (' -R 1 -m {send_size} '.format(
```

Comment:
Is this a change in response size as well?

Reply:
I have also added the -M flag to set the remote receive size as well. I do not believe it changes that response size, as UDP_STREAM and TCP_STREAM test throughput unidirectionally.